sphinx_gherkindoc/writer.py

```diff
@@ -170,15 +170,16 @@ def description(obj: BehaveModelWithDescription) -> None:
         if not isinstance(description, list):
             description = [description]
 
+        # The keyword here will be capitalized and may contain a space,
+        # so we sanitize it first.
+        role = role_name_from(f"gherkin-{obj.keyword}-description")
+        if role == "gherkin-scenario-outline-description":
+            # Scenario and Scenario Outline description roles are considered the same
+            role = "gherkin-scenario-description"
+
```

Comment:
I think there might be a cleaner way to collapse all the scenario outline foo fra roles to scenario roles, but I haven't had a chance to think hard about it and check over the test scenarios.
This gets the job done and fixes a bug, so signing off!